projects/vanilla/src/app/search/search.component.html

```diff
@@ -14,15 +14,7 @@
 
             <!-- Search form -->
             <ng-template uib-template="search-form" display="Search form" let-config>
-                <app-search-form
-                    [autocompleteSources]="config.autocompleteSources"
-                    [enableVoiceRecognition]="config.enableVoiceRecognition"
-                    [keepTab]="config.keepTab"
-                    [enableKeepFilters]="config.enableKeepFilters"
-                    [keepFilters]="config.keepFilters"
-                    [enableAdvancedForm]="config.enableAdvancedForm"
-                    [keepAdvancedSearchFilters]="config.keepAdvancedSearchFilters">
-                </app-search-form>
+                <app-search-form [autocompleteSources]="config.autocompleteSources" [searchRoute]="config.searchRoute"></app-search-form>
```

Comment:
With all these options being removed, we should remove them from the preview configurator as well.

Reply:
Also, "searchRoute" should not be part of the config, since we don't want the user to customize this.